src/main/java/org/alliancegenome/curation_api/base/BaseSQLDAO.java

```diff
@@ -47,6 +47,18 @@ public E find(String id) {
         }
     }
 
+    public E find(Long id) {
+        log.debug("SqlDAO: find: " + id + " " + myClass);
+        if(id != null) {
+            E entity = entityManager.find(myClass, id);
+            log.debug("Entity Found: " + entity);
+            return entity;
+        } else {
+            log.debug("Input Param is null: " + id);
+            return null;
+        }
+    }
+
```

Comment:
@cmpich good catch I didn't update these after we made the GeneratedCurie base class